content/participants/ttjc@aniruddh-mukherjee/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: aniruddh-mukherjee
+type: "participant"
+name: "Aniruddh Mukherjee"
+portfolio: ""
+github-link: "https://github.com/aniruddh-mukherjee"
+linkedin-link: "https://www.linkedin.com/in/aniruddha-mukherjee-64a5b1195"
+looking-for: "job, internship"
```

Comment:
Hey! @aniruddh-mukherjee Please add only one thing here either job or internship.

Reply:
Fixed it :)